testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -982,7 +974,6 @@ async fn test_roominfo_update_deduplication() -> Result<()> {
             }
         );
     }
-    assert_pending!(stream);
```

Comment:
Can you keep this, or add an assertion on the expected next value, then `assert_pending` again?

Reply:
Sure.